.github/workflows/coverage.yml

```diff
@@ -0,0 +1,40 @@
+name:                           Coverage
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: '*'
+
+jobs:
+  test:
+
+    name:                       Coverage
+    runs-on:                    ubuntu-latest
+
+    steps:
+      - uses:                   actions/checkout@v2
+
+      - uses:                   actions-rs/toolchain@v1
+        with:
+          profile:              minimal
+          toolchain:            nightly
+          override:             true
+
+      - uses:                   actions-rs/cargo@v1
```

Comment:
Interesting, there's an `actions-rs` project containing Rust-specific actions.  The Build & Test action for CI that we have just uses a vanilla `cargo test` -- is it okay to leave it as it is for now?

Reply:
Yeah, I assume that gh actions are new enough that the ecosystem is fairly dynamic. We can refactor and unify once it stabilizes.